scss/_stepped-process.scss

```diff
@@ -2,123 +2,141 @@
 // Orange stepped process
 //
 .stepped-process {
+  // scss-docs-start stepped-process-css-vars
+  --#{$prefix}stepped-process-font-size: #{$stepped-process-font-size};
+  --#{$prefix}stepped-process-font-weight: #{$stepped-process-font-weight};
+  --#{$prefix}stepped-process-max-items: #{$stepped-process-max-items};
+  --#{$prefix}stepped-process-counter: #{$stepped-process-counter};
+  --#{$prefix}stepped-process-item-padding-y: #{$step-item-padding};
+  --#{$prefix}stepped-process-item-padding-x: 0;
+  --#{$prefix}stepped-process-item-margin-end: #{$step-item-margin-end};
+  --#{$prefix}stepped-process-item-bg: #{$step-item-bg};
+  --#{$prefix}stepped-process-item-active-bg: #{$step-item-active-bg};
+  --#{$prefix}stepped-process-item-next-bg: #{$step-item-next-bg};
+  --#{$prefix}stepped-process-item-drop-shadow: none;
+  --#{$prefix}stepped-process-item-arrow-width: 0;
+  --#{$prefix}stepped-process-item-arrow-shape: #{$step-item-arrow-shape};
+  --#{$prefix}stepped-process-link-width: #{$step-link-width};
+  --#{$prefix}stepped-process-link-color: #{$step-link-color};
+  --#{$prefix}stepped-process-link-active-color: #{$step-link-active-color};
+  --#{$prefix}stepped-process-link-marker: #{$step-link-marker};
+  --#{$prefix}stepped-process-link-text-decoration: #{$step-link-text-decoration};
+  // scss-docs-end stepped-process-css-vars
```

Comment:
We didn't have the case before but shouldn't we include what's in `@include media-breakpoint-up` as well in the documentation inside some `scss-docs-end`? In the same as `stepped-process-css-vars` or in separate section explained in the docs.